### PR TITLE
Features/navigation/search

### PR DESCRIPTION
**Pull request by widmo200**
added logic to change placeholder text ;)

### CODE REVIEW

**Review by patrycja-gomza — APPROVED** [review]
Great work on this PR! 👏 I've reviewed the changes and everything looks good. The functionality is implemented smoothly and the code is well-organized.

I have just a couple of minor suggestions to further align with the design from Figma:
1. 'Search for movies' Input Height:
According to the Figma design, the input height should be 48px. Please adjust the input height to match the Figma specifications.
2. Header Elements Padding:
In the Figma design, header elements should be positioned exactly in the middle of the black bar. You have a smaller padding from the top and a larger padding from the bottom. Please adjust the padding accordingly to align with the design.

Once these tweaks are made, it will perfectly match the design guidelines.
Thanks for your hard work on this! 🚀

**Comment by widmo200** [comment]
@patrycja-gomza  @bedicooper  i wait for approve ;)

File: package-lock.json
Comment (bedicooper) [inline package-lock.json:1]: Are you sure nothing wrong happened here? That a lot of lines to delete, installing axios shouldn't have done that :)

**Review by bedicooper — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (widmo200) [reply package-lock.json:1]: please check on your local branch, it works for me and I don't know why so many lines were removed O;

File: src/Navigation/Search/searchSaga.js
Comment (bedicooper) [inline src/Navigation/Search/searchSaga.js:16]: I would use less generic name for your watcher, as all of them run on data fatching

File: package.json
Comment (bedicooper) [inline package.json:1]: When merging this file I suggest accepting incoming changes from main to aviod any bugs

**Review by bedicooper — COMMENTED** [review]
Thanks for the PR!
Slice and Saga work, the store updates with response on query 1sec after I stop typing.
As I wrote in the comments, maybe try to merge main to your branch and accept incoming changes from main on **store** and **pakcage.json** and **package-lock.json**
Just add your import and your reducer in store.js.

Maybe this way we can avoid potentialy problematic bugs if there is something important in those 18,685 line that were deleted from packege-lock.json

push to this PR after merging main and I will approve 😄

File: package-lock.json
Comment (bedicooper) [reply package-lock.json:1]: Even if it works now I don't know if something won't break later? Maybe try to restore previous state of pakcage.json and package-lock.json from main? - we have axios and saga installed so your branch should work after merging

File: src/app/store.js
Comment (widmo200) [reply src/app/store.js:1]: it's a good idea i merge main  in to this branch ;)

File: src/Navigation/Search/getSearch.js
Comment (widmo200) [reply src/Navigation/Search/getSearch.js:6]: ok ;d

File: src/Navigation/Search/searchSaga.js
Comment (widmo200) [reply src/Navigation/Search/searchSaga.js]: i changed this name  , now  this saga have : searchSaga :)

File: package.json
Comment (widmo200) [reply package.json:1]: now i  merge main into my branch and accept incoming changes ;) now is it ok ;)

**Review by bedicooper — APPROVED** [review]
Thanks!

**Pull request merged**
